packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePluginTest.kt

```diff
@@ -98,6 +102,56 @@ class AmplifyDataStorePluginTest {
         `when`(mockDataStore.getPlugin("awsDataStorePlugin")).thenReturn(mockAmplifyDataStorePlugin)
     }
 
+    @Test
+    fun test_custom_datastore_configuration() {
```

Comment:
I wonder if we should break this test down into custom config vs no custom config. I think we are inadvertantly overloading this one test right now by testing both use cases

Reply:
👍 will split the test into two